debugview/plugin.cpp

```diff
@@ -274,12 +274,13 @@ class debugview : public plugin {
 
 	
 
-	void main_loop() {
-		double lastTime = glfwGetTime();
-		glfwMakeContextCurrent(gui_window);
-		
-		while (!_m_terminate.load()) {
-
+	bool first_time = true;
+	void _p_one_iteration() override {
+		if (first_time) {
```

Comment:
Can you add a comment explaining this first_time logic?

Reply:
Done. I also `first_time -> first_iteration` to be consistent with the name of the method.